helm/prometheus/templates/service.yaml

```diff
@@ -14,7 +14,7 @@ metadata:
 {{- if .Values.service.labels }}
 {{ toYaml .Values.service.labels | indent 4 }}
 {{- end }}
-  name: {{ template "prometheus.fullname" . }}
+  name: {{ .Values.service.nameOverride | default .Release.Name }}
```

Comment:
does ` "prometheus.fullname"` has the same value of `.Release.Name`?

Reply:
I had a look at how the helper provides `prometheus.fullname` and this would be a truncated version. However, it doesn't seem possible to include the helper function as a default value. 

I'm getting parsing errors. Do you know if this is possible?

`{{ .Values.service.nameOverride | default template "prometheus.fullname" . }}`